### ISSUE TITLE
Tslint error messages in every file (PyCharm)

### ISSUE THREAD

**Issue by doomsayer2** [phovea/generator-phovea#66]
* Release number or git hash:  #64 and commit number: 784f197fea5de5d2a6e3c46ceae0e4f58058f331
* OS: Windows

### Steps to reproduce

1. Pull the latest Changes and open your IDE (PyCharm for example). 
2. Open any .ts file or file that is checked by lint. 

### Observed behavior
After pulling the latest changes, the PyCharm IDE throws Errors for linting. The errors are included in nearly every file (for me working on [TaCo](https://github.com/Caleydo/taco) currently).
![1](https://cloud.githubusercontent.com/assets/13848511/22105145/051d6fbe-de43-11e6-9dda-004cf3e0ffd2.PNG)

Here is the error Message for all files:
![2](https://cloud.githubusercontent.com/assets/13848511/22105167/1b691336-de43-11e6-8fae-ae818cd4d964.PNG)

As problem the following parts can be identified, which are marked in the IDE, once changeing to the tslint.json. Following shows just an example:
![3](https://cloud.githubusercontent.com/assets/13848511/22105203/3d9c6bec-de43-11e6-9c48-11022c756388.PNG)

A solution would be to change the keyword `false` to `true` or generally remove the other options, as tslint expects this parameter to be true, if there are options provided in the same array.

### Expected behavior
No error messages in every file once opening it.

**Comment by sgratzl** [comment]
your dependencies aren't uptodate. 

```
[yo phovea:workspace] --- depending on the setup
npm install
```

**Comment by doomsayer2** [comment]
Ok can be closed than i guess but i cant update it. As i seem to have no workspace. I just have C:\Users\taco_new and i seem to have no yeoman generator. npm install does nothing for me except saying it skips optional dependencies.

`0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'search',
1 verbose cli   'yeoman-generator' ]
2 info using npm@3.10.8
3 info using node@v6.9.1
4 silly mapToRegistry name -/all
5 silly mapToRegistry using default registry
6 silly mapToRegistry registry https://registry.npmjs.org/
7 silly mapToRegistry data Result {
7 silly mapToRegistry   raw: '-/all',
7 silly mapToRegistry   scope: null,
7 silly mapToRegistry   escapedName: null,
7 silly mapToRegistry   name: null,
7 silly mapToRegistry   rawSpec: '-/all',
7 silly mapToRegistry   spec: '-/all',
7 silly mapToRegistry   type: 'local' }
8 silly mapToRegistry uri https://registry.npmjs.org/-/all
9 info updateIndex C:\Users\fgras\AppData\Roaming\npm-cache\registry.npmjs.org\-\all\.cache.json
10 verbose correctMkdir C:\Users\fgras\AppData\Roaming\npm-cache correctMkdir not in flight; initializing
11 warn Building the local index for the first time, please be patient
12 silly update-index fetching https://registry.npmjs.org/-/all
13 verbose request uri https://registry.npmjs.org/-/all
14 verbose request no auth needed
15 info attempt registry request try #1 at 13:31:59
16 verbose request id b5b9957a2c55e528
17 http request GET https://registry.npmjs.org/-/all
18 http 200 https://registry.npmjs.org/-/all
19 verbose headers { server: 'nginx/1.4.6 (Ubuntu)',
19 verbose headers   'content-type': 'text/plain',
19 verbose headers   'last-modified': 'Thu, 19 Jan 2017 12:00:01 GMT',
19 verbose headers   etag: '"5880aa41-0"',
19 verbose headers   expires: 'Thu, 19 Jan 2017 13:27:11 GMT',
19 verbose headers   'access-control-allow-origin': '*',
19 verbose headers   'access-control-allow-methods': 'GET',
19 verbose headers   'cache-control': 'max-age=3600',
19 verbose headers   'content-length': '0',
19 verbose headers   'accept-ranges': 'bytes',
19 verbose headers   date: 'Thu, 19 Jan 2017 12:30:31 GMT',
19 verbose headers   via: '1.1 varnish',
19 verbose headers   age: '200',
19 verbose headers   connection: 'keep-alive',
19 verbose headers   'x-served-by': 'cache-hhn1534-HHN',
19 verbose headers   'x-cache': 'HIT',
19 verbose headers   'x-cache-hits': '6',
19 verbose headers   'x-timer': 'S1484829031.516497,VS0,VE0' }
20 verbose stack TypeError: Cannot read property '_updated' of undefined
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\lib\cache\update-index.js:88:26
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\lib\utils\pulse-till-done.js:20:8
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\lib\cache\caching-client.js:53:8
20 verbose stack     at f (C:\Program Files\nodejs\node_modules\npm\node_modules\once\once.js:25:25)
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:76:16
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:103:12
20 verbose stack     at f (C:\Program Files\nodejs\node_modules\npm\node_modules\once\once.js:25:25)
20 verbose stack     at CachingRegistryClient.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:297:12)
20 verbose stack     at Request._callback (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:210:14)
20 verbose stack     at Request.self.callback (C:\Program Files\nodejs\node_modules\npm\node_modules\request\request.js:187:22)
21 verbose cwd C:\Users\taco_new
22 error Windows_NT 10.0.14393
23 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "search" "yeoman-generator"
24 error node v6.9.1
25 error npm  v3.10.8
26 error Cannot read property '_updated' of undefined
27 error If you need help, you may report this error at:
27 error     <https://github.com/npm/npm/issues>
28 verbose exit [ 1, true ]
`

**Comment by thinkh** [comment]
I can confirm that error on my machine:

![tslint_pycharm](https://cloud.githubusercontent.com/assets/5851088/22107487/596950f0-de4e-11e6-9267-10a46b1ed34f.png)

Clicking on the banner brings me to the tslint.json where I get the following warnings (excerpt):

![tslintjson](https://cloud.githubusercontent.com/assets/5851088/22107489/5c2089b2-de4e-11e6-9481-70215a5621df.png)

*Steps done to solve the issue*
1. Updated the generator
2. Deleted the old tslint.json
3. Run `yo phovea:workspace` -> generates a new tslint.json
4. Open PyCharm

Result: Still having the same error.

**Comment by sgratzl** [comment]
can you check whether you have a `generator-phovea` package in our node_modules directory, if so delete it and try again.

**Comment by sgratzl** [comment]
cause the two files are identical (the init-web one and the workspace one)

**Comment by thinkh** [comment]
I have the newest generator from master 784f197fea5de5d2a6e3c46ceae0e4f58058f331 since I'm using the `npm link` approach. There is no *generator-phovea* directory in the workspace *node_modules* directory.

**Comment by sgratzl** [comment]
can you verify that in your workspace (cause I can't reproduce it)
 * tslint.json of workspace/mothertable/tslint.json and workspace/tslint.json are the same
 * workspace/package.json uses tslint: 4.3.1 (or so, 4.3.x at least)
 * tslint version is actually installed
 * pycharm uses the right directory for tslint

**Comment by thinkh** [comment]
> workspace/package.json uses tslint: 4.3.1 (or so, 4.3.x at least)
At least for me that was the pointer. I worked on some provenance-steering branches that haven't had the most recent versions in the package.json. After I merged the branches (and updated the repos package.json), I ran `yo phovea:workspace` and `npm install`. Now everything is up-to-date and the errors are gone.

I'll see the same for the taco repos.
